atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -190,23 +190,25 @@ public void close() {}
         }
     }
 
+    /**
+     * It is expected that config provides list of servers that are directly reachable and do not require special IP
+     * resolution.
```

Comment:
What happens if config does not? Do we just crash and burn?

Can we detect this case, and if so, can we log something useful?

Reply:
well we will not be able to talk to Cassandra at all, think those are logged as somewhat meaningful messages already